### PR TITLE
Convert the manage page to a PageModule.

### PR DESCRIPTION
**Pull request by shane-tomlinson**
This is a no-user facing change PR. I converted the managePage to be a PageModule so that it is consistent with other modules.

issue #2346

### CODE REVIEW

File: resources/static/pages/js/manage_account.js [hunk 1]
```diff
@@ -202,48 +203,46 @@ BrowserID.manageAccount = (function() {
...
-    var template = new EJS({ text: $("#templateManage").html() });
-    var manage = template.render({});
-    $("#hAlign").after(manage);
+      var self=this,
+          oncomplete = options.ready,
+          template = new EJS({ text: $("#templateManage").html() }),
+          manage = template.render({});
 
-    dom.bindEvent("#cancelAccount", "click", cancelEvent(cancelAccount));
+      $("#hAlign").after(manage);
```
Comment (seanmonstar) [inline resources/static/pages/js/manage_account.js]: jQuery $ slipped in, instead of bid.DOM?

File: resources/static/pages/js/manage_account.js [hunk 2]
```diff
@@ -202,48 +203,46 @@ BrowserID.manageAccount = (function() {
...
-    dom.bindEvent("button.edit", "click", startEdit);
-    dom.bindEvent("button.done", "click", cancelEdit);
-    dom.bindEvent("#edit_password_form", "submit", cancelEvent(changePassword));
+      self.click("#cancelAccount", cancelAccount);
 
-    user.checkAuthentication(function(auth_level) {
-      authLevel = auth_level;
+      self.bind("button.edit", "click", startEdit);
+      self.bind("button.done", "click", cancelEdit);
+      self.bind("#edit_password_form", "submit", cancelEvent(changePassword));
```
Comment (seanmonstar) [inline resources/static/pages/js/manage_account.js]: should this be `Module.submit` instead?

**Comment by seanmonstar** [comment]
looks good besides my comments

File: resources/static/pages/js/manage_account.js (hunk 1 above)
Comment (shane-tomlinson) [inline resources/static/pages/js/manage_account.js]: I changed off of the $'s to dom.\* calls. Thanks for pointing this out.

File: resources/static/pages/js/manage_account.js (hunk 2 above)
Comment (shane-tomlinson) [inline resources/static/pages/js/manage_account.js]: Great catch. Changed it.

**Pull request merged**